### DISCUSSION TITLE
Why are my "sketch" and "inpaint sketch" tabs effectively broken?

### DISCUSSION THREAD

**Discussion by afbagwell** [vladmandic/sdnext#737] [Q&A] [answered]
I've been using auto1111 and now this fork for a few months, and these two tabs have never worked either on my previous PC or my latest one.

First, the inpaint tab is great. I use it all the time, even on scaled up images. But the moment I try to send something of any size to sketch or inpaint sketch, I get incredible lag and freezes, and images often fail to load (white box displayed with a little frowny face in the upper-left). Forget trying to actually use the tools to do anything much less click the generate button. The only way to fix the problem is restart both the UI and server. This happens even on a vanilla install with no extra extensions.

My desktop is Win11, 64GB RAM and a 4090. Using the latest Chrome browser, but the problem shows up in others too.

Does anyone else experience this? And if you have, do you know a fix? I've searched the github discussion boards and reddit for answers but the most I have so far is some people get some lag on images bigger than 1024x1024, but they are usually running with a system much less powerful. I expect the lack of complaints is because most people don't have this problem.

**Comment by gruevy** [comment, marked as answer]
Only works in Firefox. Not sure why exactly, but I have Firefox installed just to use that.

**Reply by vladmandic** [reply]
can you create issue for this if one doesn't exist. discussions are great for questions (and discussions :) ), but if something is broken and there is no known issue, its easy for things to fall off my radar if there is no issue created.

**Reply by gruevy** [reply]
Sure, I'll see if I can find the reasons people gave for it and create an issue. It was the same in automatic1111, too, and IIRC it was a gradio thing

**Reply by afbagwell** [reply]
Well that explains it. I don't have FF but did try it in Edge without any luck. Guess I'm installing another browser.
Thanks for the tip!

**Reply by gruevy** [reply]
FYI it looks like setting your resolution to 100% instead of 125% or 150% or whatever can fix it too

**Reply by vladmandic** [reply]
that explains why i couldn't reproduce. used to have 27" 4k monitor where i had to use resolution scalling, now on 34" and no issues :(

**Reply by afbagwell** [reply]
Well... this is unexpected. I installed the latest Firefox and I can't even get past the loading screen. 
Edit: nvm... FF doesn't like my VPN making connections for outside resources. Thinks its a MITM attack
